src/main/java/org/opentripplanner/middleware/controllers/api/OtpRequestProcessor.java

```diff
@@ -135,10 +143,10 @@ private static String proxy(Request request, spark.Response response) {
 
     /**
      * Process plan response from OTP. Store the response if consent is given. Handle the process and all exceptions
-     * seamlessly so as not to affect the response provided to the requester.
+     * seamlessly so as not to affect the response provided to the requester. Returns false if there was an error.
```

Comment:
Might be better to just move that last sentence to an `@return`.

Reply:
Added in 19fb070